webrender/res/brush_image.glsl

```diff
@@ -26,6 +27,16 @@ flat varying vec2 vSelect;
     #define IMAGE_SOURCE_MASK_FROM_COLOR    2
 #endif
 
+struct ImageBrush {
+    RectWithSize rendered_task_rect;
+};
+
+ImageBrush fetch_image_primitive(int address) {
+    vec4 data = fetch_from_resource_cache_1(address);
+    RectWithSize rendered_task_rect = RectWithSize(data.xy, data.zw);
+    return ImageBrush(rendered_task_rect);
```

Comment:
let's avoid returning with a struct constructor, no need to bring up those GLSL driver bugs from the dead :)

Reply:
Fixed